examples/calimero-login/src/calimeroSdk.js

```diff
@@ -1,8 +1,8 @@
 import { CalimeroSdk } from "calimero-auth-sdk";
 
 export default CalimeroSdk.init({
-    shardId: "slim-shardy-calimero-testnet",
-    walletUrl: "https://localhost:1234",
-    calimeroUrl: "https://api.development.calimero.network",
-    calimeroWebSdkService: "https://api.development.calimero.network",
+    shardId: process.env.REACT_APP_CALIMERO_SHARD_ID,
+    walletUrl: process.env.REACT_APP_WALLET_ENDPOINT_URL,
+    calimeroUrl: process.env.REACT_APP_CALIMERO_ENDPOINT_URL,
+    calimeroWebSdkService: process.env.REACT_APP_ALIMERO_WEB_SDK_SERVICE_URL,
```

Comment:
```suggestion
    calimeroWebSdkService: process.env.REACT_APP_CALIMERO_WEB_SDK_SERVICE_URL,
```
do we really need these 2

Reply:
websdkservice not used anywhere so gonna remove it